### DISCUSSION TITLE
Is Micronaut-Security components relevant in Micronaut Serverless(Lambda)  ?

### DISCUSSION THREAD

**Discussion by subash89** [micronaut-projects/micronaut-core#10989] [Q&A]
Is micronut token validators, authentication providers (micronaut security components) ? 
Seems like in my MicronautRequestHandler class, HttpRequest object injected is null.  So seems like event HttpRequest(or @RequestScope) is also not relevant in micronaut server less

**Comment by subash89** [comment]
For example, I had to micronaut.security.filter.enabled=false to make a serverless-application request working.(We have custom Auth fetchers, and authentication providers). May be that's why SecurityFilter getting loaded and engaged.

**Comment by sdelamo** [comment]
Micronaut Security works with Micronaut Lambda. If you want to develop a Micronaut Lambda using controllers, use the handlers specified here:  https://micronaut-projects.github.io/micronaut-aws/4.6.0/guide/#requestHandlers

see guides such as: https://guides.micronaut.io/latest/mn-application-aws-lambda-java-runtime-gradle-java.html
